components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/main/resources/Actions.yaml

```diff
@@ -189,6 +191,69 @@ paths:
                 $ref: '#/components/schemas/Error'
 
   /actions/{actionType}/{actionId}:
+    get:
+      tags:
+        - Actions
+      operationId: getActionByActionId
+      summary: |
+        Retrieve Action information by given Id.
+      description: |
+        This API provides the capability to retrieve the action by action Id.<br>
+          <b>Scope required:</b> <br>
+              * internal_action_mgt_view
+      parameters:
+        - name: actionType
+          in: path
+          description: Name of the Action Type.
+          required: true
+          schema:
+            enum:
+              - preIssueAccessToken
+              - preUpdatePassword
+              - preUpdateProfile
+              - preRegistration
+              - authentication
+        - name: actionId
+          in: path
+          description: Id of the Action.
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ActionResponse'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '501':
+          description: Not Implemented
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```

Comment:
This incorporates a doc update for the actions api. @Thisara-Welmilla please track this

Reply:
Will be tracked with https://github.com/wso2/product-is/issues/21036